digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/queries.scala

```diff
@@ -250,6 +250,12 @@ object Queries {
     """.as[Int].list
   }
 
+  def getMunicipalitiesInfo: Seq[(Int, String)] = {
+    sql"""
+      select id, name_fi from municipality
+    """.as[(Int, String)].list
+  }
```

Comment:
Already exist   def getMunicipalitiesNameAndIdByCode(codes: Set[Int]): List[(Int, String)] = {
    val filter = if (codes.nonEmpty) {"where id in (" + codes.mkString(",") + ")" } else ""

    sql"""
      select id, name_fi from municipality
      #$filter
    """.as[(Int, String)].list
  } on municipalityDAO

The method bottom **getMunicipalitiesWithoutAhvenanmaa** could be moved to municipalityDAO as well

Reply:
That method receives a parameter, The one that was created doesn't